### PR TITLE
Add next steps to console instructions

### PR DESCRIPTION
**Pull request by dorbin**
Hopefully to resolve Jake's difficulty, expressed in his friction log.

### CODE REVIEW

**Comment by dorbin** [comment]
@duftler 
@demobox
@jtk54

**Review by jtk54 — APPROVED** [review]

**Review by duftler — APPROVED** [review]
LGTM.

File: k8s/spinnaker/scripts/install/provision-spinnaker.md [hunk 1]
```diff
@@ -84,18 +84,27 @@ expose it via a secure domain behind the [Identity-Aware Proxy](https://cloud.go
...
+accessed it via port forwarding or made it available over the public
+internet, you'll use this same console to manage your Spinnaker instance.
 
-```bash
-~/click-to-deploy/k8s/spinnaker/scripts/manage/update_console.sh
-```
+You can open this console by navigating to the Kubernetes Application on the
+[Applications](https://console.developers.google.com/kubernetes/application?project={{project-id}})
+view. The application's *Next Steps* section contains the relevant links and
+operator instructions.
```
Comment (demobox) [inline k8s/spinnaker/scripts/install/provision-spinnaker.md:96]: "instructions for Spinnaker admins."?

File: k8s/spinnaker/scripts/install/provision-spinnaker.md [hunk 2]
```diff
@@ -84,18 +84,27 @@ expose it via a secure domain behind the [Identity-Aware Proxy](https://cloud.go
...
-When you want to manage Spinnaker in the future, you can always locate your Spinnaker installation
-by navigating to the newly registered Kubernetes Application via the [Applications](https://console.developers.google.com/kubernetes/application?project={{project-id}}) view.
+* Use [Halyard](https://www.spinnaker.io/reference/halyard/) to further
+configure Spinnaker
+* Add provider accounts
+* Upgrade Spinnaker
+* Add more operators
 
-The application's *Next Steps* section contains the relevant links and operator instructions.
+To start managing Spinnaker:
```
Comment (demobox) [inline k8s/spinnaker/scripts/install/provision-spinnaker.md:106]: "To start managing Spinnaker, refresh this console:"?

File: k8s/spinnaker/scripts/install/provision-spinnaker.md [hunk 3]
```diff
@@ -84,18 +84,27 @@ expose it via a secure domain behind the [Identity-Aware Proxy](https://cloud.go
...
+[Applications](https://console.developers.google.com/kubernetes/application?project={{project-id}})
+view. The application's *Next Steps* section contains the relevant links and
+operator instructions.
 
-### Ongoing Management
+You can...
 
-When you want to manage Spinnaker in the future, you can always locate your Spinnaker installation
-by navigating to the newly registered Kubernetes Application via the [Applications](https://console.developers.google.com/kubernetes/application?project={{project-id}}) view.
+* Use [Halyard](https://www.spinnaker.io/reference/halyard/) to further
```
Comment (demobox) [inline k8s/spinnaker/scripts/install/provision-spinnaker.md:100]: We also have helper scripts to add provider accounts, so we many not want to point only at Halyard here

**Comment by demobox** [comment]
@dorbin FYI added a bunch of minor comments after this was merged

**Pull request merged**